.github/workflows/linter.yml

```diff
@@ -0,0 +1,27 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  pull_request:
+    branches: [main]
```

Comment:
I think we can use macro: `$default-branch`.
The same for DEFAULT_BRANCH.
See for details: 
- https://github.blog/changelog/2020-07-22-github-actions-better-support-for-alternative-default-branch-names/
- https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/sharing-workflows-with-your-organization